package.json

```diff
@@ -43,7 +44,7 @@
     "verify-travis-appveyor": "^3.0.0"
   },
   "scripts": {
-    "install": "prebuild-install || node-gyp rebuild",
```

Comment:
After merge, I'll continue working on using `prebuildify` instead of `prebuild` and `prebuild-install`.

Reply:
👍 Folks, follow #482 for progress on prebuildify.